### ISSUE TITLE
Upgrade Python version

### ISSUE THREAD

**Issue by BACCMED** [henon/Python.Included#57]
I'm working on a project that uses Python.Included

I have to use apython version upper than 3.7

Is there a possibility to upgrade python version ?

Thanks

**Comment by henon** [comment]
the latest nuget uses pythong 3.11

**Comment by mikewindlake** [comment]
Is there a way to select the python version?

**Comment by henon** [comment]
Yes, if you use the Python.Deployment nuget you can install any python version as shown in the examples.
